build.sh

```diff
@@ -59,7 +58,7 @@ function run {
 	update_news_next_ver "$ROOT_BRANCH" "$DATE" "$NEXT_VERSION"
 	verify_changes
 	commit_news_next_ver "$NEXT_VERSION"
-	push_branches "$ROOT_BRANCH"
+	push_branches "$ROOT_BRANCH" "$RELEASE_BRANCH" "$TAG_NAME"
```

Comment:
The only reason to separate this out is that if the NEWS file (which is hand edited) is in some way malformed such that `update_news_next_ver` doesn't work correctly, the build will never be pushed.